### DISCUSSION TITLE
What is the status on the "coming soon" training scripts and steps for LLaVA 1.6?

### DISCUSSION THREAD

**Discussion by SanDiegoDude** [haotian-liu/LLaVA#1223] [General]
Hi there,  I've got a captioning project I've been forced to use GPTV for as LLaVA1.5 is just not cutting it. LLaVA 1.6 gets us a lot closer, and I feel with tuning could even outperform GPTV for the same given task, however the only references I've found to training 1.6 so far is a paywalled script and an accompanying YT video. As much as I appreciate Trelis for figuring out the details (and appears to be the first and only to do so, at least publicly), it'd be nice if we could have steps for the whole community to follow from the devs themselves.  Anybody from the haotian-liu team have a rough ETA for when we can expect the script?

(apologies if this was covered already, I searched and didn't find anything)

**Comment by lightmatmul** [comment]
please update

**Comment by poudelb1** [comment]
Still no updates??
